app/src/main/java/org/mozilla/focus/topsites/TopSites.kt

```diff
@@ -152,7 +152,8 @@ private fun TopSiteFaviconCard(topSite: TopSite) {
                     horizontalAlignment = Alignment.CenterHorizontally
                 ) {
                     Text(
-                        text = topSite.url.getRepresentativeCharacter(),
+                        text = if (topSite.title.isNullOrEmpty()) topSite.url.getRepresentativeCharacter()
+                        else topSite.title?.get(0).toString(),
```

Comment:
Can formatting be improved here?
The `else` case appearing in line with `Text`'s named properties looks a bit strange. Maybe using curly braces would help?

Reply:
Done